WordPress/Classes/ViewRelated/Post/Geolocation/PostGeolocationView.m

```diff
@@ -56,6 +62,8 @@ - (void)layoutSubviews
 
     self.mapView.frame = CGRectMake(0.0, 0.0, width, mapHeight);
     self.addressLabel.frame = CGRectMake(labelX, mapHeight, labelWidth, addressLabelHeight);
+    CGSize chevronSize = self.chevron.frame.size;
+    self.chevron.frame = CGRectMake(labelWidth-chevronSize.width, mapHeight+((addressLabelHeight-chevronSize.height)/2), chevronSize.width, chevronSize.height);
```

Comment:
For improved readability, maybe compose the x and y values in variables, add spaces around the operators and, (I know this is super nitpicky) make the denominator a double instead of an int for type matching?  Something like:
```
CGFloat x = labelWidth - chevronSize.width;
CGFloat y = mapHeight + ((addressLabelHeight - chevronSize.height) / 2.0);
self.chevron.frame = CGRectMake(x, y, chevronSize.width, chevronSize.height);
```